### PR TITLE
update codeowners to add present authors and editors

### PR DESCRIPTION
**Pull request by decentralgabe**

### CODE REVIEW

**Comment by decentralgabe** [comment]
Looks like both @dlongley and @David-Chadwick need to be added to the repo before the CODEOWNERS file is valid

**Comment by David-Chadwick** [comment]
Not sure what is going to happen to me since I am about to lose
      my crossword email address
    kind regards
    David
    
    On 15/03/2023 17:52, Gabe wrote:
    
    
      
      Looks like both @dlongley
        and @David-Chadwick need to be added to
        the repo before the CODEOWNERS file is valid
      —
        Reply to this email directly, view it on GitHub, or unsubscribe.
        You are receiving this because you were mentioned.Message
          ID: ***@***.***>
      [
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/w3c/vc-data-model/pull/1067#issuecomment-1470493042",
"url": "https://github.com/w3c/vc-data-model/pull/1067#issuecomment-1470493042",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

File: CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-*       @msporny @kdenhartog
+*       @msporny @dlongley @David-Chadwick @OR13 @selfissued @decentralgabe @awoie
```
Comment (msporny) [inline CODEOWNERS:1]:
```suggestion
*       @msporny @OR13 @selfissued @decentralgabe @awoie
```

Reducing to current list of Editor's listed on the spec.

**Review by dlongley — APPROVED** [review]

**Review by brentzundel — APPROVED** [review]

**Comment by msporny** [comment]
Maintenance, multiple reviews, changes requested and made, just documents reality (Editors are CODEOWNERS), merging.

**Pull request merged**